akka-persistence/src/main/scala/akka/persistence/journal/PersistencePluginProxy.scala

```diff
@@ -136,8 +131,57 @@ final class JournalProxy(config: Config) extends Actor with Stash with ActorLogg
       if (targetAtThisNode && address != selfAddress)
         becomeIdentifying(address)
     case InitTimeout ⇒
-    case msg ⇒
-      targetJournal.forward(msg)
+
+    case msg @ WriteMessages(messages, _, actorInstanceId) ⇒
+      implicit val ec = context.system.dispatcher
+      val sender = context.sender()
+      val f = ask(targetJournal, msg)(1.second)
+      f onComplete {
+        case Success(v) ⇒ sender ! v
+        case Failure(t) ⇒
+          sender ! WriteMessagesFailed(t)
+          messages foreach {
+            case a: AtomicWrite       ⇒ a.payload.foreach { p ⇒ sender ! WriteMessageFailure(p, t, actorInstanceId) }
+            case r: NonPersistentRepr ⇒ sender ! LoopMessageSuccess(r.payload, actorInstanceId)
+          }
+      }
+    case msg @ DeleteMessagesTo(_, toSeqNr, _) ⇒
+      implicit val ec = context.system.dispatcher
+      val sender = context.sender()
+      val f = ask(targetJournal, msg)(2.seconds)
```

Comment:
and use a longer default, e.g. 10 seconds (some journals are slow when starting up in testing mode, e.g. embedded cassandra)